src/FsCheck/Gen.fs

```diff
@@ -313,6 +313,20 @@ module Gen =
     [<CompiledName("ListOf")>]
     let listOfLength n arb = sequence [ for _ in 1..n -> arb ]
 
+    ///Generates a random permutation of the given sequence.
+    //[category: Creating generators]
+    [<CompiledName("Shuffle")>]
+    let shuffle xs =
+        gen {
+            let h = 922337203
```

Comment:
Does `bounds`or `max` communicate the _origin_ of the value itself?

Can that origin be communicate by the value's _name_, or would it be possible to communicate the information via other means?